### PR TITLE
feat: add chainpatrol monitoring

### LINKED ISSUE #322: feat: scan for malicious link with ChainPatrol
We should use ChainPatrol API to scan for malicious link within proposals, similar as the BlockAid integration and using a config var for the API key.

### PR DESCRIPTION
**Pull request by ChaituVR**
Fixes #322 

- Also added a new folder monitoring to add these services
- Update blockaid to use API key from env
- Small refactoring

## How to test:
- Try to create proposals using spam links for example `daolido.org`
- Try to create proposals using genuine links

### CODE REVIEW

File: src/writer/proposal.ts [hunk 1]
```diff
@@ -104,22 +104,19 @@ export async function verify(body): Promise<any> {
...
   try {
     const content = `
       ${msg.payload.name || ''}
       ${msg.payload.body || ''}
       ${msg.payload.discussion || ''}
     `;
-
     if (await isMalicious(content)) {
-      blockaidBlockedRequestsCount.inc({ space: space.id });
+      chainpatrolBlockedRequestsCount.inc({ space: space.id });
```
Comment (wa0x6e) [inline src/writer/proposal.ts:114]: Maybe move this one directly inside the chainpatrol helper

Comment (ChaituVR) [reply src/writer/proposal.ts:114]: no `space.id` in helper 🙈 is it worth to pass?

**Review by bonustrack — COMMENTED** [review]
We don't need a folder "spamPrevention", also can you remove the harcoded API key from Blockaid, and use a config var instead?

**Comment by wa0x6e** [comment]
Having a folder is good IMO, keep things structured

**Comment by bonustrack** [comment]
If you want a folder call it "monitoring", this is not about spam

File: src/writer/proposal.ts (hunk 1 above)
Comment (ChaituVR) [reply src/writer/proposal.ts:114]: there is no `space` name inside `proposal` data 😄

Comment (ChaituVR) [reply src/writer/proposal.ts:114]: will pass `space` as seperate param for now

**Review by wa0x6e — APPROVED** [review]
utAck

**Pull request merged**